presentation/voca/src/main/java/com/hilingual/presentation/voca/component/VocaEmptyCard.kt

```diff
@@ -0,0 +1,77 @@
+package com.hilingual.presentation.voca.component
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import com.hilingual.core.designsystem.theme.HilingualTheme
+import com.hilingual.presentation.voca.R
+
+enum class VocaEmptyCardType(
+    val text: String,
+    @DrawableRes val imageRes: Int
+) {
+    NOTADD(
+        text = "아직 단어가 추가되지 않았어요",
+        imageRes = R.drawable.img_word_and
+    ),
+    NOTSEARCH(
+        text = "검색 결과가 없습니다",
+        imageRes = R.drawable.img_search_and
+    )
```

Comment:
P1: 
```suggestion
    NOT_ADD(
        text = "아직 단어가 추가되지 않았어요",
        imageRes = R.drawable.img_word_and
    ),
    NOT_SEARCH(
        text = "검색 결과가 없습니다",
        imageRes = R.drawable.img_search_and
    )
```
enum 네이밍 할 때 띄어쓰기는 _로 해보는거 어때요~~??